### PR TITLE
Enabled Prometheus for Jobservice

### PR DESCRIPTION
**Pull request by ninjadq**
* Add prom server on jobservice
* Enabeld configs in templates
* Enabeld jobservice metrics in nginx

Signed-off-by: DQ <dengq@vmware.com>

### CODE REVIEW

**Comment by ninjadq** [comment]
image

**Review by bitsf — APPROVED** [review]

**Review by steven-zou — CHANGES_REQUESTED** [review]
please do changes

File: src/vendor/github.com/gocraft/work/worker_pool.go [hunk 1]
```diff
@@ -199,6 +199,11 @@ func (wp *WorkerPool) Start() {
 	wp.periodicEnqueuer.start()
 }
 
+// GetPoolID return the pool ID
+func (wp *WorkerPool) GetPoolID() string {
+	return wp.workerPoolID
+}
+
```
Comment (heww) [inline src/vendor/github.com/gocraft/work/worker_pool.go:206]: DO NOT EDIT files in vendor manually, please update them through go.mod and go.sum.

**Review by heww — CHANGES_REQUESTED** [review]
DO NOT EDIT files in vendor manually, please update them through go.mod and go.sum.

**Review by steven-zou — APPROVED** [review]
Please make sure the code coverage diff is in a reasonable scope.

**Review by heww — APPROVED** [review]

**Pull request merged**